runtime/runtime/src/actions.rs

```diff
@@ -271,7 +271,9 @@ pub(crate) fn action_function_call(
         }
         // Update action result with the abort error converted to the
         // transaction runtime's format of errors.
-        let action_err: ActionError = ActionErrorKind::FunctionCallError(err.into()).into();
+        let action_err: ActionError =
+            ActionErrorKind::FunctionCallError(crate::conversions::function_call_error::cvt(err))
```

Comment:
Is method `cvt` a standard as well, can we continue using `into` here which is more readable?